upb/json/printer.c

```diff
@@ -208,18 +208,30 @@ static size_t fmt_bool(bool val, char* buf, size_t length) {
   return n;
 }
 
-static size_t fmt_int64(long val, char* buf, size_t length) {
+static size_t fmt_int32(long val, char* buf, size_t length) {
   size_t n = _upb_snprintf(buf, length, "%ld", val);
   CHKLENGTH(n > 0 && n < length);
   return n;
 }
 
-static size_t fmt_uint64(unsigned long long val, char* buf, size_t length) {
+static size_t fmt_uint32(unsigned long long val, char* buf, size_t length) {
   size_t n = _upb_snprintf(buf, length, "%llu", val);
   CHKLENGTH(n > 0 && n < length);
   return n;
 }
 
+static size_t fmt_int64(long val, char* buf, size_t length) {
```

Comment:
Int64 should be `long long` (on 32-bit `long` will only be 32 bits).

Reply:
done